usr.sbin/cron/cron/externs.h

```diff
@@ -49,14 +49,14 @@ extern char *tzname[2];
 
 #if !defined(POSIX) && !defined(ATT)
 /* classic BSD */
-extern	time_t		time();
-extern	unsigned	sleep();
-extern	struct tm	*localtime();
-extern	struct passwd	*getpwnam();
+extern	time_t		time(void);
+extern	unsigned	sleep(void);
+extern	struct tm	*localtime(void);
+extern	struct passwd	*getpwnam(void);
 extern	int		errno;
-extern	void		perror(), exit(), free();
-extern	char		*getenv(), *strcpy(), *strchr(), *strtok();
-extern	void		*malloc(), *realloc();
+extern	void		perror(void), exit(void), free(void);
+extern	char		*getenv(void), *strcpy(void), *strchr(void), *strtok(void);
+extern	void		*malloc(void), *realloc(void);
```

Comment:
this feels confusing.
it makes sense that it's possible, but I've never seen it like this.

Reply:
Also, "extern"  is used implicitly for functions declarations.

Reply:
I'd delete all of these, and then include the proper .h file to get them instead...